Build failed  on x86_64-darwin:

```
In file included from core/src/tbox/tbox/src/tbox/platform/file.c:41:
core/src/tbox/tbox/src/tbox/platform/posix/file.c:697:34: error: use of undeclared identifier 'UTIME_OMIT'
            else ts[0].tv_nsec = UTIME_OMIT;
                                 ^
core/src/tbox/tbox/src/tbox/platform/posix/file.c:699:34: error: use of undeclared identifier 'UTIME_OMIT'
            else ts[1].tv_nsec = UTIME_OMIT;
                                 ^
core/src/tbox/tbox/src/tbox/platform/posix/file.c:700:19: error: call to undeclared function 'utimensat'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
            ok = !utimensat(AT_FDCWD, path, ts, 0);
                  ^
core/src/tbox/tbox/src/tbox/platform/posix/file.c:715:38: error: use of undeclared identifier 'UTIME_OMIT'
                else ts[0].tv_nsec = UTIME_OMIT;
                                     ^
core/src/tbox/tbox/src/tbox/platform/posix/file.c:717:38: error: use of undeclared identifier 'UTIME_OMIT'
                else ts[1].tv_nsec = UTIME_OMIT;
                                     ^
core/src/tbox/tbox/src/tbox/platform/posix/file.c:718:23: error: call to undeclared function 'futimens'; ISO C99 and later do not support implicit function declarations [-Wimplicit-function-declaration]
                ok = !futimens(tb_file2fd(file), ts);
                      ^
6 errors generated.
```

https://logs.ofborg.org/?key=nixos/nixpkgs.343185&attempt_id=e2479983-f76a-4d28-a737-c1bc19a2ce6b

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -18916,8 +18916,8 @@ with pkgs;
 
   webdis = callPackage ../development/tools/database/webdis { };
 
-  xmake = callPackage ../development/tools/build-managers/xmake {
-    inherit (darwin.apple_sdk.frameworks) CoreServices;
+  xmake = darwin.apple_sdk_11_0.callPackage ../development/tools/build-managers/xmake {
+    inherit (darwin.apple_sdk_11_0.frameworks) CoreServices;
   };
 
   xc3sprog = callPackage ../development/embedded/xc3sprog { };

